assets/js/translator.js

```diff
@@ -223,6 +223,8 @@ if(modeEnabled('translation')) {
                 $('#fileInput').show();
                 $('div#fileName').hide();
                 $('div#docTranslation').fadeIn('fast');
+                $('#detect').prop('disabled', true);
+                $('#srcLangSelect').children('option[value=' + 'detect' + ']').attr('disabled', true);
```

Comment:
I followed the style as used across the code, but yeah, we are looking only for a constant `string`, I'd change it now!